run_cumulus_task.py

```diff
@@ -19,12 +28,24 @@ def run_cumulus_task(task_function, cumulus_message, context):
 
     message_adapter_disabled = os.environ.get('CUMULUS_MESSAGE_ADAPTER_DISABLED')
 
-    if message_adapter_disabled:
-        return task_function(cumulus_message, context)
+    if message_adapter_disabled is True:
```

Comment:
What I meant by my earlier comment was `os.environ.get('CUMULUS_MESSAGE_ADAPTER_DISABLED') == 'true'`, not explicitly testing if the resulting value was `True`.  I'm pretty sure that `message_adapter_disabled` cannot ever be `True` the way that line 29 is written right now.

Reply:
oh, got it! I'l fix that